bundles/org.openhab.binding.webexteams/src/main/java/org/openhab/binding/webexteams/internal/WebexTeamsHandler.java

```diff
@@ -0,0 +1,315 @@
+        message.setFile(attach);
+        logger.debug("Sending message with attachment to default room ({})", config.roomId);
+        return sendMessage(message);
+    }
+
+    /**
+     * Send a message to a specific room
+     * 
+     * @param roomId roomId of the room to send to
+     * @param msg markdown text string to be sent*
+     * @return <code>true</code>, if sending the message has been successful and
+     *         <code>false</code> in all other cases.
+     */
+    public boolean sendRoomMessage(String roomId, String msg) {
+        Message message = new Message();
+        message.setRoomId(roomId);
+        message.setMarkdown(msg);
+        logger.debug("Sending message to room {}", roomId);
+        return sendMessage(message);
+    }
+
+    /**
+     * Send a message to a specific room
+     * 
+     * @param roomId roomId of the room to send to
+     * @param msg markdown text string to be sent
+     * @param attach URL of the attachment
+     * 
+     * @return <code>true</code>, if sending the message has been successful and
+     *         <code>false</code> in all other cases.
+     */
+    public boolean sendRoomMessage(String roomId, String msg, String attach) {
+        Message message = new Message();
+        message.setRoomId(roomId);
+        message.setMarkdown(msg);
+        message.setFile(attach);
+        logger.debug("Sending message with attachment to room {}", roomId);
+        return sendMessage(message);
+    }
+
+    /**
+     * Sends a message to a specific person, identified by email
+     * 
+     * @param personEmail email address of the person to send to
+     * @param msg markdown text string to be sent
+     * @return
+     */
+    public boolean sendPersonMessage(String personEmail, String msg) {
+        Message message = new Message();
+        message.setToPersonEmail(personEmail);
+        message.setMarkdown(msg);
+        logger.debug("Sending message to {}", personEmail);
+        return sendMessage(message);
+    }
+
+    /**
+     * Sends a message to a specific person, identified by email
+     * 
+     * @param personEmail email address of the person to send to
+     * @param msg markdown text string to be sent
+     * @param attach URL of the attachment*
+     * @return
+     */
+    public boolean sendPersonMessage(String personEmail, String msg, String attach) {
+        Message message = new Message();
+        message.setToPersonEmail(personEmail);
+        message.setMarkdown(msg);
+        message.setFile(attach);
+        logger.debug("Sending message to {}", personEmail);
+        return sendMessage(message);
+    }
+
+    /**
+     * Sends a <code>Message</code>
+     * 
+     * @param msg the <code>Message</code> to be sent
+     * @return <code>true</code>, if sending the message has been successful and
+     *         <code>false</code> in all other cases.
+     */
+    private boolean sendMessage(Message msg) {
+        client.sendMessage(msg);
+        return true;
```

Comment:
Problem, this method always returns true ?

Reply:
Fixed... checking for exceptions now.